js/src/elements/form-behavior.ts

```diff
@@ -691,6 +680,16 @@ export function navigateAfterSubmit(href: string, navigationStep: string | null)
 				location.href = href;
 			}
 			break;
+		case 'hash-reload':
```

Comment:
Would you be open to setting the valid strings explicitly in this function's signature?

I'm thinking something like:
```ts
export function navigateAfterSubmit(
    href: string,
    navigationStep: 'follow' | 'hash-reload' | 'replace' | 'reload' | null
) {
```

This way, as we're invoking `navigateAfterSubmit`, we can get intellisense as to which options are available.

Reply:
(Feel free to disregard or table this for later)

Reply:
I think that's fine, had to add it as a type though.